pages/lessons/[lesson]/[slug].tsx

```diff
@@ -131,30 +125,42 @@ const Lesson: React.FC<LessonProps> = ({
 export default Lesson
 
 export const getStaticPaths = async () => {
-  const files = fs.readdirSync(path.join('lessons'))
-  const paths = files.map((filename) => ({
+  const trackDir = path.join('lessons')
+
+  const topicDirectories = fs.readdirSync(trackDir)
```

Comment:
This is a cool idea -- so once we have more tracks, I'd assume we're rename `projects` folder to say `Track 1: NFTs` and the UI would change accordingly.

Though one challenge would be to present spaces and special characters if we don't use a single word for a track name.